services/121-service/src/events/events.service.ts

```diff
@@ -20,8 +20,11 @@ import { ScopedUserRequest } from '@121-service/src/shared/scoped-user-request';
 import { UserService } from '@121-service/src/user/user.service';
 import { UserType } from '@121-service/src/user/user-type-enum';
 
+// TODO: REFACTOR: Turn this into an interface instead of a type, for simplicity, and think of a better name than "LogEntity".
 type LogEntity = Partial<RegistrationViewEntity> & {
+  // TODO: REFACTOR: Would the registrationId not always be in the RegistrationViewEntity? If not, can we make it so (enforce it)? If not, just call it registrationId to be more clear?
```

Comment:
Because you can select which 'registration view' fields you want to return when using the paginate package. See for example 'select' here in the url of example on the paginate package [readme](https://github.com/ppetzold/nestjs-paginate?tab=readme-ov-file#endpoint). So there is guarantee which fields return